python/lsst/analysis/ap/zooniverseCutouts.py

```diff
@@ -187,18 +187,15 @@ def get_exposures(instrument, detector, visit):
         result = []
         for i, source in enumerate(data.to_records()):
             try:
-                center = lsst.geom.SpherePoint(
-                    source["ra"], source["decl"], lsst.geom.degrees
-                )
+                center = lsst.geom.SpherePoint(source["ra"], source["decl"], lsst.geom.degrees)
```

Comment:
Why this change? I agree a single line looks cleaner but I'm not sure how close it is to column length limits.

Reply:
I think somehow black or another auto-formatter was run on this code with line length of 80 (I think during the initial package setup). All of these were just trying to revert them to 110 (they fit fine). These changes are on a separate commit.

Reply:
Ah, wait, scratch that: Meredith had run black and isort on this and must have had a shorter line length setting: https://github.com/lsst/analysis_ap/commit/526036e39af0dc06575065237cf7c44d79a4fd06